linter_exclusions.yml

```diff
@@ -37,6 +37,16 @@ iot dps enrollment-group update:
     auth_type_dataplane:
       rule_exclusions:
       - no_parameter_defaults_for_update_commands
+iot hub state migrate:
+  parameters:
+    orig_resource_group_name:
+      rule exclusions:
+      - parameter_should_not_end_in_resource_group
+iot hub state migrate:
```

Comment:
`iot hub state migrate` is defined twice, you should put both parameters and their exclusions in the same bucket.